### DISCUSSION TITLE
How do I see stopped containers?

### DISCUSSION THREAD

**Discussion by Luxvao** [rancher-sandbox/rancher-desktop#3225] [Q&A] [answered]
I tried `docker container list`, but it only shows runnning containers. How do I list all containers? If name a container "test" and I do `docker stop test` and then check `docker container list` it's not there. I can start it again after and it pops up. Is this intended?
Thanks

**Comment by jandubois** [comment, marked as answer]
You can use `docker container list --all` to include stopped containers.

A shorter command would be `docker ps` and `docker ps -a`.
